README.md

```diff
@@ -1,62 +1,16 @@
-# NEAR Protocol Stateless Validation available for public preview
+# NEAR Stake Wars IV: Attack of Transactions
 
-NEAR Protocol is preparing a major update of the core protocol architecture that takes scalability to the next level. This update is called stateless validation and it has ample impact on how the network operates internally. We invite everyone to join an early access network that runs on this new architecture, StatelessNet, and run a stateless validator node or test the network throughput and scalability.
+NEAR Protocol is preparing a major update of the core protocol architecture that takes scalability to the next level. This update is called stateless validation, and it has ample impact on how the network operates internally. We invite everyone to join an early access network that runs on this new architecture, StatelessNet, and run a stateless validator node or test the network throughput and scalability.
```

Comment:
We only invite mainnet validators to be statelessnet validators but everyone to bug bounty and traffic generation.

Reply:
I don't think we should limit being a validator only to existing validators. OFK current validators are our main audience, but we don't plan to restrict anyone trying a new role

Reply:
There is a lot of confusion around this, please see this thread for hopefully the latest status:
https://near.zulipchat.com/#narrow/stream/422293-pagoda.2Fcore.2Fstake-wars-iv/topic/Who's.20eligible.20for.20rewards/near/419242181

Reply:
I re-checked, it's covered in bounty page. Anyone can still run a validator node, but only current mainnet validators may receive the reward